### DISCUSSION TITLE
installation problem

### DISCUSSION THREAD

**Discussion by Panda4234** [winstxnhdw/lc-hax#26] [Q&A]
i can't download those packages while lauch 'requirements.bat',is there another way?

**Comment by winstxnhdw** [comment]
Please describe why you are unable to download the packages? In fact, the `requirements.bat` doesn’t install anything.

**Comment by Panda4234** [comment]
> Please describe why you are unable to download the packages? In fact, the `requirements.bat` doesn’t install anything.

wasn't it restore the nuget package?i think?i don't know,it said can't find those package from the score

**Comment by winstxnhdw** [comment]
If you have .NET 8.0 installed, that shouldn't be a problem.

**Comment by Panda4234** [comment]
> If you have .NET 8.0 installed, that shouldn't be a problem.

i am installed,i add another scorce to fix that

i think 'requirments.bat' could work now,but lunch.bat got  '0x000002C247921C60'

**Comment by Panda4234** [comment]
oh,i see how it works now

**Comment by winstxnhdw** [comment]
Great! Maybe you can describe how you solve it for others with the same issue as yourself.
